Navaids/VOR_UK.txt

```diff
@@ -14,7 +14,6 @@ DTY 116.400 N052.10.48.510 W001.06.49.640 ; Daventry (DME)
 DCS 115.200 N054.43.18.880 W003.20.26.300 ; Dean Cross (DME)
 DET 117.300 N051.18.14.410 E000.35.50.190 ; Detling (DME)
 DVR 114.950 N051.09.45.440 E001.21.32.720 ; Dover (DME)
-GAM 112.800 N053.16.53.280 W000.56.49.790 ; Gamston (DME)
```

Comment:
Sorry @rishab-alt this was my mistake in specifying the issue, the VOR has been removed but the DME remains, hence why it was removed from the AD chart. The definition in this file was updated by the general VOR update already, so can we undo this change, and make this issue only the removal of the heli stand.

Sorry for the confusion!

Reply:
Okay  LOL